scripts/init.sh

```diff
@@ -11,11 +11,18 @@ pushd  polkadot-sdk
 git checkout snowbridge
 popd
 popd
-
 ln -sf ../polkadot-sdk polkadot-sdk
 
-echo "Setting up submodules"
-git submodule update --init --recursive || true
+echo "Checkout lodestar Snowfork fork"
+pushd ..
+  if [ ! -d "lodestar" ]; then
+    git clone https://github.com/ChainSafe/lodestar
+  fi
+  pushd lodestar
+    git checkout $LODESTAR_VERSION
+  popd
```

Comment:
```suggestion
  pushd lodestar
    git fetch
    git checkout $LODESTAR_VERSION
  popd
```

I tested this locally. We need to do a fetch here otherwise git can't find the tag